examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+# Basic Python CI With Cloud Source Repositories and Cloud Build
+
+## Overview
+This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
+
+The basic steps we need to follow are:
+1. Create a new project in Cloud Source Repository and clone it to your machine
+2. Create your code and unit tests
+3. Create a Python Cloud Builder (more below)
+4. Create a cloud build trigger (more below)
+5. Create a cloudbuild.yaml file that executes your tests using your python cloud builder.
+
+## 1. Create a new project in Cloud Source Repository
+* Go to https://source.cloud.google.com/
+* Click 'Add repository'
+* Choose 'Create new repository'
+* Specify a name, and your project name
+* Follow the instructions to 'git clone' the empty repo to your workstation and use it like you would a git repo
```

Comment:
Can you provide some more guidance around this? The idea is they should take this sample and clone it into their project, right? Or maybe I am misunderstanding.

Reply:
Yes, this is a bit of an artifact of where it lived vs. where it currently lives.  I will write a few sentences around how to use this better.